packpack

```diff
@@ -201,7 +201,6 @@ chcon -Rt svirt_sandbox_file_t ${PACKDIR} ${SOURCEDIR} ${BUILDDIR} \
 # Start Docker
 #
 set -ex
-docker pull ${DOCKER_REPO}:${DOCKER_IMAGE}
```

Comment:
> 'docker run' pull the image automatically, so w/o explicit pull command
> it will just use the local one (if any)

It is true, but there is a nuance. The image is updated by the `docker pull` command if it was updated in the registry. Without the explicit `docker pull` command it is not performed.

It allows to update the image with, for example, APT/YUM repository URI update if the old mirror is gone (such as https://github.com/packpack/packpack-docker-images/pull/117). This way we can fix CI in many repositories by just updating the image, without manual action to update the image on CI runners.

I think we should keep the existing behavior (with explicit `docker pull`) by default and provide an option (an environment variable) to skip the `docker pull` command.

Reply:
Yes, it's reasonable. I'll fix.

Reply:
Done

Reply:
Thanks!